experimental/LieAlgebras/src/LieAlgebras.jl

```diff
@@ -70,17 +71,22 @@ export LieSubalgebra
 export LieAlgebraModule, LieAlgebraModuleElem
 export LieAlgebraModuleHom
 export LinearLieAlgebra, LinearLieAlgebraElem
-
+export RootSystem
+export SimpleLieAlgebra, SimpleLieAlgebraElem
 export abelian_lie_algebra
```

Comment:
```suggestion
export RootSystem
export SimpleLieAlgebra, SimpleLieAlgebraElem

export abelian_lie_algebra
```
keep empty line between Uppercase and lowercase exports